### ISSUE TITLE
Don't show placeholder when hovering draggable source item

### ISSUE THREAD

**Issue by nico281** [reppners/ngx-drag-drop#11]
Hi, how are you?
I have a question regarding the drag effect.
When you start to drag an item, the item continue being visible and gives the illusion of having an additional item on the list. the same is with only one item, you have the effect of drop on top or below the same item you started to grab. do you think is a good behavior? i think it could be a behavior bug.
Cheers

**Comment by nico281** [comment]
Thx for your response my friend!!
The thing a don´t understand is the dndPlaceholderRef. you need to use it to get access to the event id and that´s the responsible to the awkward behave. i will share a picture
![pic](https://user-images.githubusercontent.com/15875218/39014767-b1a54ca6-43f1-11e8-8f6b-09faaefd1b16.png)

**Comment by reppners** [comment]
You need the element with `dndPlaceholderRef` only when you want to implement a sortable list. On drop you will get the index in the drop event so your logic can insert it at the right position. You can also style the `dndPlaceholderRef` element any way you want it to look. Or not use it at all. The only consequence will be that it will not be sortable.

**Comment by nico281** [comment]
the thing is that i want to change the behavior of the dndPlaceholderRef.  You use a div to move across the list to reference the drop place. but if you have one element for example and try to move that element that div appears or on top or below that single element and i want only to appear if you pass over another element for example.

**Comment by reppners** [comment]
I think I got you now 😁 

Let me rephrase your requirements to make sure we're on the same page.

------------------------------------------------------------------

Don't show the `dndPlaceholderRef` when all of the following conditions are true
* the current drop index/position of a dragged item is equal to the original index/position
* the dragged item is originally inside the same drop zone that is currently dragged over

------------------------------------------------------------------

That's very valid point and would be a good enhancement but it may not be super easy to implement.

PRs or ideas for how to implement welcome :) Can't promise any timeframe to get this implemented as I'm rather busy at the moment.

**Comment by ChristofFritz** [comment]
Stale issue.
